framework/src/modules/reward/schemas.ts

```diff
@@ -46,3 +46,27 @@ export const configSchema = {
 	},
 	required: ['tokenIDReward', 'offset', 'distance', 'brackets'],
 };
+
+export const getDefaultRewardAtHeightRequestSchema = {
+	$id: '/reward/endpoint/getDefaultRewardAtHeightRequest',
```

Comment:
`$id` has leading `/` in here but this is not the case in other places in this PR.

Reply:
I check the code, and it seems this is quite inconsistent everywhere. Ill create another issue for refactoring

Reply:
Created https://github.com/LiskHQ/lisk-sdk/issues/7173